ansible/roles/blackbox_exporter/templates/blackbox_exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus Blackbox Exporter
+
+[Service]
+User=nobody
+Group=nogroup
+EnvironmentFile=-/etc/default/blackbox_exporter
+PermissionsStartOnly=true
+ExecStartPre=/sbin/setcap 'cap_net_raw+ep' {{ blackbox_exporter_path }}/blackbox_exporter
```

Comment:
AFAIK, IPPROTO_ICMP is granted using EGID via `net.ipv4.ping_group_range` sysctl on Linux (group vector is not enough), I know nothing about MacOSX.
I suggest to continue the discussion at prometheus/blackbox_exporter#147